packages/docs-reanimated/docs/advanced/useHandler.mdx

```diff
@@ -91,7 +83,9 @@ Each of the event worklets will receive the following parameters when called:
 
 #### `dependencies` <Optional />
 
-Changes in `dependencies` array cause `useHandler` hook to receive updated values during rerender of the wrapping component. This matters when, for instance, worklet uses values dependent on the component's state.
+An optional array of dependencies.
+
+Only relevant when using Reanimated [without the Babel plugin on the Web.](/docs/guides/web-support/#web-without-the-babel-plugin)
 
 This array may contain:
 
```

Comment:
We can remove this and the following paragraph too

Reply:
Whole info about contents of array, si?